### PR TITLE
JENA-1714: Migrate 'migrate' packages

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: jena-arq/src/main/java/org/apache/jena/sparql/algebra/OpLib.java [hunk 1]
```diff
@@ -18,31 +18,18 @@
...
 package org.apache.jena.sparql.algebra;
 
-import org.apache.jena.sparql.algebra.op.* ;
-
-// NB Operations take and return nulls for convenience.
+import org.apache.jena.sparql.core.Quad;
 
 public class OpLib
 {
-    public static Op sub(Op1 op) { return op==null ? null : op.getSubOp() ; }
```
Comment (ajs6f) [inline jena-arq/src/main/java/org/apache/jena/sparql/algebra/OpLib.java]: Did these guys just not get any use?

Comment (afs) [reply jena-arq/src/main/java/org/apache/jena/sparql/algebra/OpLib.java]: No.

Comment (ajs6f) [reply jena-arq/src/main/java/org/apache/jena/sparql/algebra/OpLib.java]: Got it, makes sense.

File: jena-base/src/main/java/org/apache/jena/atlas/lib/StreamOps.java [hunk 2]
```diff
@@ -34,25 +39,32 @@
...
     public static <X> X element(Collection<X> collection) {
         return first(collection.stream());
     }
 
+    /** Debug : print stream.
+     * This operation prints the whole stream at the point it is used, 
+     * and then returns a new stream of the same elements.
+     */ 
     public static <X> Stream<X> print(Stream<X> stream) {
         stream = stream.map(item -> { System.out.println(item); return item; });
```
Comment (ajs6f) [inline jena-base/src/main/java/org/apache/jena/atlas/lib/StreamOps.java]: Isn't this line just `stream = stream.peek(System.out::println)`?

Comment (ajs6f) [reply jena-base/src/main/java/org/apache/jena/atlas/lib/StreamOps.java]: Oops-- just realized this isn't part of the PR, sorry!

**Pull request merged**